easybuild/easyconfigs/p/POV-Ray/POV-Ray-3.7.0.8-gompi-2019b.eb

```diff
@@ -0,0 +1,60 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu, NTUA
+# Authors::   Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'POV-Ray'
+version = '3.7.0.8'
+
+homepage = 'http://www.povray.org/'
```

Comment:
There is a `https` version of this url, but visiting gives me `Error code: SSL_ERROR_UNSUPPORTED_VERSION`. So, I am not sure changing this to `https` is the right solution.

Reply:
And the https version works now.